app/components/Form/ReadOnlyArrayFieldTemplate.tsx

```diff
@@ -0,0 +1,41 @@
+import { ArrayFieldTemplateProps } from "@rjsf/core";
+
+const ReadOnlyArrayFieldTemplate = (props: ArrayFieldTemplateProps) => {
+  return (
+    <>
```

Comment:
Would it be better to `<>` to `<div>` so we can have fields of array field template vertically?
![Screenshot 2023-07-10 at 4 03 32 PM](https://github.com/bcgov/cas-cif/assets/52381975/4cea83f4-e8c2-4b87-bab0-27f7d34db3e9)